demo/anonymous.html

```diff
@@ -50,6 +50,9 @@ <h1 class="brand-heading"><i class="icon-conversejs"></i>Converse.js</h1>
 </body>
 
 <script>
+        /* This Source Code Form is subject to the terms of the Mozilla Public
+         * License, v. 2.0. If a copy of the MPL was not distributed with this
+         * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
```

Comment:
I think this should be public domain or CC-0 or something similar, I wouldn’t be comfortable with putting that kind of license on a “configuration file”.  Same for other occurences of `converse.initialize()`.

Reply:
Good point. I'll put the Unlicense on these as it's short enough.